src/pages/en/guides/troubleshooting.md

```diff
@@ -11,16 +11,17 @@ Astro provides several different tools to help you troubleshoot and debug your c
 
 Here are some common error messages you might see in the terminal, what they might mean, and what to do about them.
 
-### Transform failed with X error
+### Transform failed with "Unexpected `export`"
 
-This message often appears due to a current limitation in Astro requiring your import and export statements to be at the top of your `.astro` file.
+A current limitation in Astro is that `export` statements (other than `getStaticPaths()`) are not supported in `.astro` files.
```

Comment:
Noticing that this still doesn’t mention `Prop`:
```suggestion
A current limitation in Astro is that `export` statements (other than `getStaticPaths()` and `Props`) are not supported in `.astro` files.
```

Reply:
I removed this entirely. This limitations was actually removed in https://github.com/withastro/compiler/pull/463. I opened https://github.com/withastro/astro/pull/4003 to provide a better error hint in context for `export` hoisting issues, since `export` _can_ cause reference errors. We'll keep an eye out for this and can put that error message here if users have trouble.